ui/src/shared/utils/resourceToTemplate.ts

```diff
@@ -0,0 +1,94 @@
+import _ from 'lodash'
+import {getDeep} from 'src/utils/wrappers'
+import {Task, Label} from 'src/types/v2'
+import {ITemplate, TemplateType} from '@influxdata/influx'
+
+const CURRENT_TEMPLATE_VERSION = '1'
+
+const blankTemplate = {
```

Comment:
maybe this and the `blankTaskTemplate` could be a constant functions instead. Just to avoid the small chance of someone writing code in the future that creates a template and mutates it—that could create a very difficult to track down bug 

```
const blankTemplate = () => ({
  meta: {version: '0.1.0'},
  content: {data: {}, included: []},
  labels: [],
})
```

Reply:
👍